cloudinit/config/cc_disk_setup.py

```diff
@@ -982,7 +982,7 @@ def mkfs(fs_cfg):
 
         # File systems that support the -F flag
         if overwrite or device_type(device) == "disk":
-            fs_cmd.append(lookup_force_flag(fs_type))
+            fs_cmd += filter(None, [lookup_force_flag(fs_type)])
```

Comment:
Using `+=` and `filter` on a single item list seems more complex than necessary. What do you think about something like:

```suggestion
            force_flag = lookup_force_flag(fs_type)
            if force_flag:
                fs_cmd.append(force_flag)
```

It's a couple more lines, but I think it's more explicit about what we're actually doing here.